src/guide/command-line-options.md

```diff
@@ -104,6 +104,12 @@ vendor/bin/phpunit --coverage-xml=build/coverage/coverage-xml --log-junit=build/
 infection.phar --coverage=build/coverage
 ```
 
+### `--skip-initial-test-suite`
+
+If you have already run the test suite to ensure it is passing, and you are providing coverage using `--coverage` then you can use this option to increase performance by not running the entire test suite again.
+
+Note that is it extremely important that you ensure the test suite is passing when using this option, otherwise test failures would appear like caught mutations and those mutations may be reported incorrectly.
```

Comment:
```suggestion
<p class="tip">Note that is it extremely important that you ensure the test suite is passing when using this option, otherwise test failures would appear like caught mutations and those mutations may be reported incorrectly.</p>
```